### ISSUE TITLE
Error running: clang-format: Unknown command line argument '-sort-includes'

### ISSUE THREAD

**Issue by chenai** [google/vim-codefmt#64]
I try to use `:FormatCode`, it throws:
```
Error formatting file: ERROR(ShellError): Error running: clang-format -style file -assume-filename /Users/shiyang/test.c -lines 1:29 -cursor 134 -sort-includes
clang-format: Unknown command line argument '-sort-includes'.  Try: 'clang-format -help'
clang-format: Did you mean '-style'?
```
my clang-format version:
```
➜  ~  clang-format -version
clang-format version 3.8.0 (tags/google/testing/2015-07-31)
```

**Comment by dbarnett** [comment]
Looks like the sort-includes feature was added _after_ 3.8 was released (added in r248367, 3.8 released in r242222). So the version check in #59 wouldn't work for revisions in that range.

As a workaround, could you just update to a newer clang-format?

**Comment by malcolmr** [comment]
I think r242222 was updating the trunk version to mark it as 3.8.0 (3.8.0 hasn't actually been released yet).

However, it looks like -sort-includes is now on-by-default anyway (r253202), so we should probably just undo #59.
